Rules/CommonScripts/ChatCommands/PlayerCommands.as

```diff
@@ -0,0 +1,231 @@
+#include "ChatCommand.as"
+
+class ArcherCommand : ChatCommand
```

Comment:
A fair bit of duplicated stuff, I would also move most functionality to a common class (or just to a unique `!swap` command that uses a whitelist for classes to swap to).